### PR TITLE
Initial doltgres docs and replication instructions

### PR DESCRIPTION
**Pull request by zachmu**

### CODE REVIEW

File: content/products/doltgres/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Doltgres is pre-alpha
+ 
+Doltgres is still in very active development and many required features are missing. Check back
+often for progress. You can also follow our [blog](https://www.dolthub.com/blog/) for updates, where
+we publish DoltgreSQL blogs every few weeks.
+
+Here are some of the DoltgreSQL blogs we've published so far:
+
+- [Announcing DoltgreSQL](https://www.dolthub.com/blog/2023-11-01-announcing-doltgresql/)
```
Comment (Hydrocharged) [inline content/products/doltgres/README.md:44]: Is this something that we're planning to keep updated as we release new blogs? Or just occasionally update this with some of our best/favorites?

File: content/products/doltgres/replication-from-postgres.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+After enabling logical replication, create a publication for the tables you want replicated to dolt,
+as well as a replication slot for the DoltgreSQL replica to connect to. From the `psql` shell, run
+the following commands as an admin user:
+
+```bash
+% export DBNAME=postgres
+% export TABLE=t1
+% export SLOTNAME=doltgres_slot
+% psql "dbname=$DBNAME replication=database" -c "CREATE PUBLICATION $SLOTNAME FOR TABLE $TABLE"
```
Comment (Hydrocharged) [inline content/products/doltgres/replication-from-postgres.md]: Super small but I'd add semicolons to the ends of these statements. The `CREATE PUBLICATION` ones under this section too.

File: content/products/doltgres/running-server.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: Running the DoltgreSQL Server
+---
+
+Start the DoltgreSQL server with the sql-server command:
+
+```bash
+% doltgres sql-server
```
Comment (Hydrocharged) [inline content/products/doltgres/running-server.md]: I'd probably just remove this, or if we want to keep it, put it second.

File: content/products/doltgres/running-server.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  host: localhost
+  port: 5432
+  read_timeout_millis: 28800000
+  write_timeout_millis: 28800000
+
+data_dir: /var/doltgres/dbs
+
+cfg_dir: .doltcfg
+
+% doltgres --config config.yaml
```
Comment (Hydrocharged) [inline content/products/doltgres/running-server.md]: I think we should put this in its own block, and the call using the config in a separate block. Looks a bit confusing with the `cat` call.

File: content/products/doltgres/running-server.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+# Configuration options
```
Comment (Hydrocharged) [inline content/products/doltgres/running-server.md]: Seems like this should come before the config example.

File: content/products/doltgres/replication-from-postgres.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+To automatically create a Dolt commit for every replicated transaction, set the
+`dolt_transaction_commit` parameter in the `behavior` stanza to true. This will let you see diffs
+between any two replicated transactions on the primary.
+
+In every othr respect, a replication server is a normal Doltgres server, which means you can make
+other connections to it and run other queries (including queries that may interfere with
+replication).
+
+```yaml
```
Comment (Hydrocharged) [inline content/products/doltgres/replication-from-postgres.md:84]: Stick this before the previous paragraph.

**Review by Hydrocharged — APPROVED** [review]
A few comments but looks good overall! I'll add sections on supported statements in a separate PR.

File: content/products/doltgres/README.md (hunk 1 above)
Comment (zachmu) [reply content/products/doltgres/README.md:44]: For the short term, every blog. Eventually we'll have enough for a best-of selection

**Pull request merged**